src/controller.ts

```diff
@@ -139,6 +139,21 @@ export class AugurNodeController {
     this.logger.addLogger(logger);
   }
 
+  public systemDbVersion(): number {
+    return DB_VERSION;
+  }
+
+  public highestUserDbVersion(): number {
+    const version = 0;
+    try {
+      // only interested in mainnet databases
+      return getHighestDbVersion(this.databaseDir, "augur-1-");
```

Comment:
Only works for mainnet?

Reply:
I don't we create if users have to resync testnets.

Reply:
updated to use augur networkId instead of hardcoding mainnet networkId.